registration/migrations/0007_fix_invalid_domain_names.py

```diff
@@ -38,14 +38,14 @@ def fix_invalid_domain_names(apps, schema_editor):
     BetaTestApplication = apps.get_model('registration', 'BetaTestApplication')
     OpenEdXInstance = apps.get_model('instance', 'OpenEdXInstance')
 
-    BetaTestApplication.objects.update(subdomain=clean_domain_name('subdomain'))
-    OpenEdXInstance.objects.update(
-        internal_lms_domain=clean_domain_name('internal_lms_domain'),
-        internal_lms_preview_domain=clean_domain_name('internal_lms_preview_domain'),
-        internal_studio_domain=clean_domain_name('internal_studio_domain'),
-        internal_discovery_domain=clean_domain_name('internal_discovery_domain'),
-        internal_ecommerce_domain=clean_domain_name('internal_ecommerce_domain'),
-    )
+    # BetaTestApplication.objects.update(subdomain=clean_domain_name('subdomain'))
```

Comment:
This was erroring out, and I just wanted to get past it and see what the issue is. I will undo this. 
That said I think we can clean up out migrations a lot. We are the only users of Ocim so we can gain a lot by squashing migrations and cleaning up old python migrations. It will make some things faster, like tests. We will not need to run this operations again on production or otherwise.

Reply:
Ah no problem! I'll consider this change as a WIP.

>That said I think we can clean up out migrations a lot. We are the only users of Ocim so we can gain a lot by squashing migrations and cleaning up old python migrations. It will make some things faster, like tests. We will not need to run this operations again on production or otherwise.

I think we can use the `squashmigrations` feature from Django. 
It'll probably make the tests a little bit faster too. :)